core/objects/src/main/kotlin/app/aaps/core/objects/wizard/BolusWizard.kt

```diff
@@ -86,9 +91,12 @@ class BolusWizard @Inject constructor(
     @Inject lateinit var processedDeviceStatusData: ProcessedDeviceStatusData
 
     var timeStamp: Long
+    // var phoneNumber: String
```

Comment:
review https://github.com/nightscout/AndroidAPS/pull/3507/files and cleanup the code